.github/workflows/shinyapps_deploy.yml

```diff
@@ -9,7 +9,7 @@ on:
   push:
     branches:
       - main
-      - develop
+      - develop*
```

Comment:
I think only the branch named as develop will trigger deployment to -testing instance. Should we keep it as develop, instead of develop*?